webrender/src/device/gfx/buffer.rs

```diff
@@ -268,93 +272,120 @@ impl<B: hal::Backend> InstancePoolBuffer<B> {
         self.last_update_size = 0;
     }
 
-    fn deinit(self, device: &B::Device, heaps: &mut Heaps<B>) {
+    pub(super) fn deinit(self, device: &B::Device, heaps: &mut Heaps<B>) {
         self.buffer.deinit(device, heaps);
     }
+
+    fn free(&self) -> usize {
```

Comment:
let's rename it to something that is *not* a verb, since this isn't acting on anything

Reply:
Renamed it to `space_left` https://github.com/szeged/webrender/pull/300/commits/d5c90a5aaa484f69dfdbbddacc74449a7ffa6b60#diff-5dbe2f57bb9273f9a363bcb71ec6834eR279